libpod/runtime_ctr.go

```diff
@@ -67,6 +93,10 @@ func (r *Runtime) NewContainer(ctx context.Context, rSpec *spec.Spec, options ..
 		}
 	}
 
+	ctr.valid = true
+	ctr.state.State = ContainerStateConfigured
```

Comment:
This an runtime should be up with the rest of the stuff from newContainer

Reply:
I don't follow, what's the proposed change?